avionics/.vscode/settings.json

```diff
@@ -1,81 +0,0 @@
-{
```

Comment:
Should add .vscode to gitignore now that its removed so that git doesnt try to track it every time.

Reply:
Done